### PR TITLE
This is needed for the field rendering

### PR DESCRIPTION
**Pull request by blakeparkinson**
putting it here for now. Will need to move it somewhere more appropriate

### CODE REVIEW

File: models/players.js [hunk 1]
```diff
@@ -223,7 +243,7 @@ function mongoInsertPlayers(rb_team_id, league, team_document){
   console.log("inserting into the DB");
   db.open(function(err, db){
     db.collection("players").update({team_id: rb_team_id},
-    {$set: {team_id: team_document["team_id"], team_name: team_document["team_name"], league: league, last_updated: new Date().toISOString().slice(0, 19).replace('T', ' '), players: team_document["players"]}},
+    {$set: {team_id: team_document["team_id"], team_name: team_document["team_name"], league: league, last_updated: new Date().toISOString().slice(0, 19).replace('T', ' '), starters: team_document.starters, bench: team_document.bench}},
```
Comment (kelseyedmodo) [inline models/players.js]: Can we be consistent with either team_document["starters"] or team_document.starters with this list?

File: models/players.js [hunk 2]
```diff
@@ -30,6 +35,20 @@ var returnPlayers = function (players, rb_team_id, res, league){
   }
 }
 
+var formatEvenOdds = function(players){
```
Comment (kelseyedmodo) [inline models/players.js]: I don't know much about mobile vs. web displays, but does this restrict us too much with small screens to force the 2 columns?

**Comment by kelseyedmodo** [comment]
otherwise lgtm :sheep:

File: models/players.js (hunk 2 above)
Comment (blakeparkinson) [inline models/players.js]: I was just gonna do it with css, if the screen is a certain width, then I will disable the float that the class "right" will have

**Comment by blakeparkinson** [comment]
@kelseyedmodo 

kulgee, I updated this PR to fix a bug with the import script where it was not inserting the right team_id. Can you review?

**Comment by kelseyedmodo** [comment]
nice. looks good. ran all nba teams with no rate limiting, too!

**Pull request merged**